### ISSUE TITLE
Problem editing config in Zowe next

### ISSUE THREAD

**Issue by plavjanik** [zowe/zowe-cli#1335]
I have latest version of the Zowe CLI next on Mac:
```
zowe --version                                                                                                                                       
7.0.0-next.202203021947
```

I am trying to edit global config:
```
zowe config edit --gc
``` 

But a different application than `vi` is opened. The same problem happens when I do following:
```
ZOWE_EDITOR=code zowe config edit --gc
```

The "Live Video Delay" is opened instead.

![image](https://user-images.githubusercontent.com/7644758/157641007-fb88a0ab-e507-41f0-be9b-c05439271b19.png)

What can I do to diagnose the problem?

**Comment by plavjanik** [comment]
You are right. The Live Video Delay has registered as the default application for `.json` and I did not know about it. Changing it to VS Code fixed my problem. And it is cool that you open the default app associated with `.json` 👍🏼

**Comment by t1m0thyj** [comment]
> It seems like we only respect the value of `ZOWE_EDITOR` when running in a non-graphical environment (e.g., an SSH terminal), but we should always respect it.

This has been fixed in #752 and the latest `@zowe/cli@next`.

**Comment by plavjanik** [comment]
Thank you for fixing it :) What is the latest version? For me it is `7.0.0-next.202203101634` and I do not see a different behavior.
